standalone-reactui/src/test/java/io/atlasmap/standalone/E2ETest.java

```diff
@@ -0,0 +1,143 @@
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.atlasmap.standalone;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assume.assumeTrue;
+
+import java.io.File;
+import java.nio.file.FileSystems;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardWatchEventKinds;
+import java.nio.file.WatchEvent;
+import java.nio.file.WatchKey;
+import java.nio.file.WatchService;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import io.atlasmap.core.ADMArchiveHandler;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ActiveProfiles("test")
+@ContextConfiguration(classes = { Application.class, CorsConfiguration.class, SecurityConfiguration.class })
+public class E2ETest {
+
+    private static final String DLDIR = System.getProperty("user.dir") + File.separator + "target";
+
+    @LocalServerPort
+    int port;
+
+    ChromeDriver driver;
+
+    @Before
+    public void before() {
+        String driverPath = System.getProperty("webdriver.chrome.driver");
+        assumeTrue(driverPath != null && !driverPath.isEmpty());
+
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--headless", "--disable-gpu", "--no-sandbox", "--disable-setuid-sandbox");
+        Map<String, Object> prefs = new HashMap<>();
+        prefs.put("download.default_directory", DLDIR);
+        options.setExperimentalOption("prefs", prefs);
+        driver = new ChromeDriver(options);
+    }
+
+    @After
+    public void after() {
+        if (driver != null) {
+            driver.quit();
+        }
+    }
+
+    @Test
+    public void test() throws Exception {
```

Comment:
@atlasmap/runtime @atlasmap/ui, here is a e2e test performing - please take a quick look in case you want to add e2e test in the future